DFC.ServiceTaxonomy.Cypher/Helpers/Neo4JHelper.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DFC.ServiceTaxonomy.Cypher.Configuration;
+using Neo4j.Driver;
+
+namespace DFC.ServiceTaxonomy.Cypher.Helpers
+{
+    public class Neo4JHelper : INeo4JHelper, IDisposable
```

Comment:
Can we replace this with `NeoGraphDatabase` (especially with the multi-drivers incoming).